example/lib/main.dart

```diff
@@ -70,12 +70,16 @@ class MyHomePage extends StatelessWidget {
       body: Column(
         children: <Widget>[
           const Expanded(child: CropperWidget()),
-          Row(
-            children: const [
-              UndoButton(),
-              Expanded(child: RotationSlider()),
-              BoxShapeButton(),
-              RatioButton(),
+          Column(
+            children: [
+              Row(
+                children: const [
+                  UndoButton(),
+                  BoxShapeButton(),
+                  RatioButton(),
+                ],
+              ),
+              const RotationSlider()
```

Comment:
In mobile devices, when the slider was in a row with others, you could not skip less than 2 degrees at a time, because of limited space. this way.